src/pipeline_manager.py

```diff
@@ -156,7 +191,7 @@ def visualize(pipeline_name, image_dir=None, single_image=None, n_files=16, show
 
     images_with_drawn_boxes = pipeline.fit_transform(data)
     for img in images_with_drawn_boxes:
-        basewidth = 600
+        basewidth = 500
```

Comment:
Can we move this magic number somewhere?

Reply:
@jakubczakon it's Neptune magic

Reply:
ok